protostar/git/git_helpers.py

```diff
@@ -50,7 +50,7 @@ def run_git(*args: Union[str, Path], cwd: Path):
 
 def has_user_git_credentials():
     try:
-        subprocess.run(["git", "config", "user.name"], check=True, **SHARED_KWARGS)
+        subprocess.run(["git", "config", "user.name"], check=True, **SHARED_KWARGS)  # type: ignore
```

Comment:
why do we need this?

Reply:
pyright suddenly exploded with 170 lines of type errors here

Reply:
[same problem](https://app.circleci.com/pipelines/github/software-mansion/protostar/777/workflows/b07068d9-dcb8-4a7f-b9b9-c705e70c74b1/jobs/5215)